examples/rcbilling-demo/src/util/PurchasesLoader.ts

```diff
@@ -32,7 +36,7 @@ const loadPurchases: LoaderFunction<IPurchasesLoaderData> = async ({
 
     const [customerInfo, offerings] = await Promise.all([
       purchases.getCustomerInfo(),
-      purchases.getOfferings(),
+      purchases.getOfferings({ currency: currency || "USD" }),
```

Comment:
Should we default to `undefined`? That way we allow the SDK to use the default.

Reply:
Changing it!